helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ZkHelixClusterVerifier.java

```diff
@@ -335,6 +352,11 @@ public B setZkAddr(String zkAddress) {
       return setZkAddress(zkAddress);
     }
 
+    public B setWaitTillVerify(int waitPeriod) {
+      _coolDown = waitPeriod;
```

Comment:
Let's just call it "waitPeriodTillVerify. Cooldown is not an obvious name.

Reply:
changed